docs/en/modes/track.md

```diff
@@ -231,14 +231,17 @@ In the following example, we demonstrate how to utilize YOLO11's tracking capabi
 
         if success:
             # Run YOLO11 tracking on the frame, persisting tracks between frames
-            results = model.track(frame, persist=True)
+            results = model.track(frame, persist=True)[0]
 
             # Get the boxes and track IDs
-            boxes = results[0].boxes.xywh.cpu()
-            track_ids = results[0].boxes.id.int().cpu().tolist()
+            if results.boxes and results.boxes.id is not None:
+                boxes = results.boxes.xywh.cpu()
+                track_ids = results.boxes.id.int().cpu().tolist()
+            else:
+                pass
```

Comment:
why empty else?

Reply:
@Borda yes good point, this should be removed @ankanpy